pmm-app/tests/ia/pages/api/templatesAPI.js

```diff
@@ -0,0 +1,45 @@
+const { I, ruleTemplatesPage } = inject();
+const assert = require('assert');
+
+module.exports = {
+  async createRuleTemplate(path) {
+    const [, content, id] = await ruleTemplatesPage.ruleTemplate.templateNameAndContent(path);
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const body = {
+      yaml: content,
+    };
+    const resp = await I.sendPostRequest('v1/management/ia/Templates/Create', body, headers);
+
+    assert.ok(resp.status === 200, `Failed to create a template with id "${id}".
+     Response message is "${resp.data.message}"`);
+
+    return id;
+  },
+
+  async clearAllTemplates() {
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const resp = await I.sendPostRequest('v1/management/ia/Templates/List', { reload: true }, headers);
+
+    if (resp.data === {}) return;
+
+    for (const i in resp.data.templates) {
+      const template = resp.data.templates[i];
+
+      // eslint-disable-next-line no-continue
+      if (template.source === 'BUILT_IN') continue;
```

Comment:
LOL, continue is old-school, you could as well do this:
`
if (template.source !== 'BUILT_IN') {
   await this.removeTemplate(template.name);
}
`

Reply:
Then you don't need to continue :)

Reply:
done, thanks